README.md

```diff
@@ -7,180 +7,211 @@ gollum -- A wiki built on top of Git
+* If you're installing from source:
+	* Optionally uninstall any previous versions of Gollum:  
+		```
+		[sudo] gem uninstall -aIx gollum
+		```
+	* Install [Bundler](http://bundler.io/).
+	* Navigate to the cloned source of Gollum.
+	* Install dependencies:  
+		```
+		[sudo] bundle install
+		```
+	* And finally:  
+		```
+		rake build
+		[sudo] gem install --no-ri --no-rdoc pkg/gollum*.gem
+		```
+
+### Markups
+
+Gollum presently ships with support for the following markups:
+* [Markdown (Kramdown flavour)](http://kramdown.gettalong.org/quickref.html)
 * [RDoc](http://rdoc.sourceforge.net/)
-* [ReStructuredText](http://docutils.sourceforge.net/rst.html) -- `easy_install docutils`
-* [Textile](http://en.wikipedia.org/wiki/Textile_(markup_language)) -- `gem install RedCloth`
-* [MediaWiki](http://www.mediawiki.org/wiki/Help:Formatting) -- `gem install wikicloth`
 
-[bundler]: http://gembundler.com/
+As denoted by [github/markup](https://github.com/github/markup/blob/master/lib/github/markups.rb), support for additional markups can easily be added by additional installation:
+* [AsciiDoc](http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/) -- `[sudo] gem install asciidoctor`
+* [Creole](http://www.wikicreole.org/wiki/CheatSheet) -- `[sudo] gem install creole`
+* [GitHub Flavored Markdown](https://help.github.com/articles/github-flavored-markdown) -- `[sudo] gem install github-markdown`
+* [MediaWiki](http://www.mediawiki.org/wiki/Help:Formatting) -- `[sudo] gem install wikicloth`
+* [Org](http://orgmode.org/worg/dev/org-syntax.html) -- `[sudo] gem install org-ruby`
+* [Pod](http://perldoc.perl.org/perlpod.html) -- requires Perl >= 5.10 (the `perl` command must be available on your command line)
+	* Lower versions should install `Pod::Simple` from CPAN.
+* [ReStructuredText](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html) -- requires python >= 2 (the `python2` command must be available on your command line)
+	* Note that Gollum will also need you to install `docutils` for your Python 2. Installation procedure can, again, vary depending on operating system and package manager.
+* [Textile](http://redcloth.org/hobix.com/textile/quick.html) -- `[sudo] gem install RedCloth`
+
+As for Markdown, Gollum ships with the `kramdown` gem. Sometimes, however, it might be useful for you if Markdown was to be rendered using another gem. As denoted by [github/markup](https://github.com/github/markup/blob/master/lib/github/markup/markdown.rb), the following gems may be used:
+* `github/markdown`
+* `redcarpet`
+* `rdiscount`
+* `maruku`
+* `kramdown`
+* `bluecloth`
+
+The thing to remember here is that `github/markup` will search for a gem to render Markdown exactly in the given order, which means that (e.g.) `redcarpet` will NOT be used if `github/markdown` is installed.
```

Comment:
`exactly in the given order` -> `in the order defined [here](https://github.com/github/markup/blob/master/lib/github/markup/markdown.rb#L6-L25)`